### DISCUSSION TITLE
[SSL: CERTIFICATE_VERIFY_FAILED] Error

### DISCUSSION THREAD

**Discussion by meetu30** [pitchmuc/adobe_analytics_api_2.0#49] [Q&A]
Hi,
I am trying to get access Token using JWT token for adobe Analytics but I am getting SSL error -
SSLError: HTTPSConnectionPool(host='ims-na1.adobelogin.com', port=443): Max retries exceeded with url: /ims/exchange/jwt (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1123)')))
Here is the API call - 
imsexchange = "https://ims-na1.adobelogin.com/ims/exchange/jwt", and payload has clinet_id, client_secret and JWT token value in the form of dictionary
response = requests.post(config["imsexchange"],headers=header_jwt,data=payload) 
Could you please provide some input? Thanks in advance.

**Comment by chenz9197** [comment]
Hi,

I encountered the same issue, did you solve this eventually? Thank you
